lazy_tensor_core/lazy_tensor_core/__init__.py

```diff
@@ -8,90 +13,84 @@
-      'grpc.http2.min_ping_interval_without_data_ms=300000',  # 5 min
-  ]
-  os.environ['TF_GRPC_DEFAULT_OPTIONS'] = ','.join(options)
+    # Setup GRPC options to correctly talk to TPU backends.
+    options = [
+        'grpc.keepalive_time_ms=60000',  # 1 min
+        'grpc.keepalive_timeout_ms=14400000',  # 4 hrs
+        'grpc.http2.max_pings_without_data=0',  # unlimited
+        'grpc.http2.min_ping_interval_without_data_ms=300000',  # 5 min
+    ]
+    os.environ['TF_GRPC_DEFAULT_OPTIONS'] = ','.join(options)
 
 
 def _set_missing_flags(flags, sets):
-  for name, defval in sets:
-    insert = True
-    for fval in flags:
-      m = re.match(r'(--)?([^=]+)', fval)
-      if m and m.group(2) == name:
-        insert = False
-        break
-    if insert:
-      flags.append('--{}={}'.format(name, defval))
-  return flags
+    for name, defval in sets:
+        insert = True
+        for fval in flags:
+            m = re.match(r'(--)?([^=]+)', fval)
+            if m and m.group(2) == name:
+                insert = False
+                break
+        if insert:
+            flags.append('--{}={}'.format(name, defval))
+    return flags
 
 
 def _setup_ltc_flags():
-  flags = os.environ.get('LTC_FLAGS', '').split(' ')
-  flags = _set_missing_flags(flags, (('ltc_cpu_enable_fast_math', 'false'),))
-  os.environ['LTC_FLAGS'] = ' '.join(flags)
+    flags = os.environ.get('LTC_FLAGS', '').split(' ')
+    flags = _set_missing_flags(flags, (('ltc_cpu_enable_fast_math', 'false'),))
+    os.environ['LTC_FLAGS'] = ' '.join(flags)
 
 
 def _set_missing_env(name, value):
-  if name not in os.environ:
-    os.environ[name] = value
+    if name not in os.environ:
+        os.environ[name] = value
 
 
 def _setup_default_env():
-  _set_missing_env('TF_CPP_MIN_LOG_LEVEL', '1')
-  _set_missing_env('TPU_HOST_BOUNDS', '1,1,1')
-  _set_missing_env('GRPC_VERBOSITY', 'ERROR')
-  if server_is_alive():
-    _set_missing_env('XRT_START_LOCAL_SERVER', '0')
+    _set_missing_env('TF_CPP_MIN_LOG_LEVEL', '1')
+    _set_missing_env('TPU_HOST_BOUNDS', '1,1,1')
+    _set_missing_env('GRPC_VERBOSITY', 'ERROR')
+    if server_is_alive():
+        _set_missing_env('XRT_START_LOCAL_SERVER', '0')
 
 
 _fd, _tmp_fname = -1, ''
 
 
 def _setup_debug_env():
-  fd, tmp_fname = tempfile.mkstemp('.ptltc', text=True)
-  _set_missing_env('LTC_FNTRACKER_FILE', tmp_fname)
-  return fd, tmp_fname
+    fd, tmp_fname = tempfile.mkstemp('.ptltc', text=True)
+    _set_missing_env('LTC_FNTRACKER_FILE', tmp_fname)
+    return fd, tmp_fname
 
 
 def _summarize_fn_tracker():
-  if not _tmp_fname:
-    return
-  from .debug.frame_parser_util import process_frames
-  process_frames(_tmp_fname)
-  os.close(_fd)
-  os.remove(_tmp_fname)
+    if not _tmp_fname:
+        return
+    from .debug.frame_parser_util import process_frames
+    process_frames(_tmp_fname)
+    os.close(_fd)
+    os.remove(_tmp_fname)
 
 
 # These needs to be called before the _LAZYC module is loaded.
```

Comment:
it looks like you moved _LAZYC import above the _setup functions, but according to the comment the _setup functions need to run before _LAZYC import.  I'm not sure what the reasons are for this or of it's still important, but could you confirm?  It might be safer to just mark these as #noqa or whatever the equivalent marking is to suppress linter warnings here

Reply:
It was done by autopep8. I'm actually not sure if flake8 would complain. Let me fix it.